docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
```

Comment:
What's the user story for this, i.e. why would someone use this over the default Knative broker?
This would probably be a good place for some introductory content, not so much about what the Kafka broker is (we could include the link from earlier here again maybe), but _why_ as a user would you want to install or use this?

Reply:
I added this https://github.com/knative/docs/pull/2740/files#diff-20631ccdd823c2921c4d6bda9b486bfaR7-R17, please, let me know if it helps.